test/kubernetes-test.sh

```diff
@@ -32,16 +32,20 @@ helm template --validate install/kubernetes/cilium \
 
 kubectl apply -f cilium.yaml
 
-while true; do
-    result=$(kubectl -n kube-system get pods -l k8s-app=cilium | grep "Running" -c)
-    echo "Running pods ${result}"
-    if [ "${result}" == "2" ]; then
+pollCiliumPods () {
+  while true; do
+      result=$(kubectl -n kube-system get pods -l k8s-app=cilium | grep "Running" -c)
+      echo "Running pods ${result}"
+      if [ "${result}" == "2" ]; then
+          echo "result match, continue with kubernetes"
+          break
+      fi
+      sleep 1
+  done
```

Comment:
Actually, we can use `until` here instead of `while true`. That'll allow us to get rid of the `break`.